apispec_webframeworks/flask.py

```diff
@@ -114,3 +153,38 @@ def path_helper(self, operations, view, **kwargs):
                     method = getattr(view.view_class, method_name)
                     operations[method_name] = yaml_utils.load_yaml_from_docstring(method.__doc__)
         return self.flaskpath2openapi(rule.rule)
+
+
+class DocumentedBlueprint(Blueprint):
+    """Flask Blueprint which documents every view function defined in it."""
+
+    def __init__(self, name, import_name, spec):
+        """
+        Initialize blueprint. Must be provided an APISpec object.
+        :param APISpec spec: APISpec object which will be attached to the blueprint.
+        """
+        super(DocumentedBlueprint, self).__init__(name, import_name)
+        self.documented_view_functions = defaultdict(list)
+        self.spec = spec
+
+    def route(self, rule, documented=True, **options):
```

Comment:
I overrode this function to include the `documented=True` kwarg, as I like declaring them explicitly.

Do you like it this way or shall I remove this?